core/utils/xml.ts

```diff
@@ -137,5 +146,11 @@ export function textToDomDocument(text: string): Document {
  */
 export function domToText(dom: Node): string {
   const oSerializer = new XMLSerializer();
-  return oSerializer.serializeToString(dom);
+  return sanitizeText(oSerializer.serializeToString(dom));
```

Comment:
Nit: please fix hungarian notation (`oSerializer` -> `serializer`) while you're at it.

Reply:
I just removed the const assignment.